designer/client/components/Fee/FeeItems.jsx

```diff
@@ -163,10 +202,14 @@ export class FeeItems extends React.Component {
               <th className="govuk-table__header" scope="col">
                 Condition
               </th>
+              <th className="govuk-table__header" scope="col">
+                Quantity
```

Comment:
This should be multiplier (optional). 

The quantity should always be assumed as 1 - since that's what the existing behaviour is. we should always "add" optional behaviour to the designer and runner to ensure backwards compatibility.